pkgs/development/compilers/nextpnr/default.nix

```diff
@@ -12,13 +12,13 @@ let
 in
 with stdenv; mkDerivation rec {
   pname = "nextpnr";
-  version = "2019.08.21";
+  version = "2019.08.27";
 
   src = fetchFromGitHub {
     owner  = "yosyshq";
     repo   = "nextpnr";
-    rev    = "c192ba261d77ad7f0a744fb90b01e4a5b63938c4";
-    sha256 = "0g2ar1z89b31qw5vgqj2rrcv9rzncs94184dgcsrz19p866654mf";
+    rev    = "cce5cb65c108140d997eb60ffd92a9fc7ad02a5c";
+    sha256 = "16yb3i3igfzyvvsry643ys6c10pz7y6dx8885qrdya7xl7frr5fs";
```

Comment:
To get this to build on macos or llvm environments it needs to be given `llvmPackages.openmp` by adding:

```
++ (lib.optional stdenv.cc.isClang llvmPackages.openmp);
```

to the `buildInputs`. With this it builds for me on macos 10.13, along with `nmigen` and `nmigen-boards`.

Reply:
thanks, added!